cmd/litcli/autopilot.go

```diff
@@ -55,19 +54,21 @@ var addAutopilotSessionCmd = cli.Command{
 			Name:     "feature",
 			Required: true,
 		},
-		cli.StringFlag{
-			Name: "channel-restrict-list",
-			Usage: "list of channel IDs that the " +
-				"Autopilot server should not " +
-				"perform actions on. In the " +
-				"form of: chanID1,chanID2,...",
+		cli.StringSliceFlag{
+			Name: "feature-channel-restrict-list",
```

Comment:
technically this rename is a backwards incompatible change... imo it is fine since I dont think anyone is using this at the command line level yet. But perhaps something we should at least include in the release notes? (btw, I think we should have a running issue per release where we keep track of notes we want to make in the release so that we dont forget cc @ViktorTigerstrom )

Reply:
can we move these to be next to `feature-config` just so that all the per-feature flags are together?

Reply:
ah - I saw @ViktorTigerstrom's suggestion now re keeping the old flags for now. 

We could do that and then add a waring and a todo that they will be removed in the release after the next one. That's definitely the more friendly way of doing things :)

Reply:
cool, I changed it to remain backwards compatible